FusionIIIT/applications/academic_information/models.py

```diff
@@ -1,6 +1,6 @@
+from applications.globals.models import ExtraInfo
 from django.db import models
 
-from applications.globals.models import ExtraInfo
 
 # Create your models here.
 
```

Comment:
You do not have to make this unnecessary change

Reply:
you said that type is not needed

Reply:
Yes but there were other fields as well

Reply:
no there is just it , specialization and type

Reply:
Add:
doctor_name
doctor_phone